project-fortis-interfaces/src/components/Admin/Streams/StreamEditor.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { DataGrid } from '../DataGrid';
+import { getColumns } from '../shared';
+import CreateStream from './CreateStream';
+import StreamParamsButtonFormatter from './StreamParamsButtonFormatter';
+import StreamStatusButtonFormatter from './StreamStatusButtonFormatter';
+import { Card, CardTitle, CardText } from 'material-ui/Card';
+import Divider from 'material-ui/Divider';
+
+class StreamEditor extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.remove = this.remove.bind(this);
```

Comment:
Same comment as above: bind is not necessary with class-level arrow methods to preserve this-context.